dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/FileSizeUtil.java

```diff
@@ -1,24 +1,38 @@
 package edu.harvard.iq.dataverse.common;
 
+import static edu.harvard.iq.dataverse.common.BundleUtil.getCurrentLocale;
+import static edu.harvard.iq.dataverse.common.BundleUtil.getStringFromBundle;
+import static java.lang.Long.numberOfLeadingZeros;
+import static java.lang.String.format;
+
 public class FileSizeUtil {
-    
-    /* This method turns a number of bytes into a human readable version
-     */
-    public static String bytesToHumanReadable(long v) {
-        return bytesToHumanReadable(v, 1);
-    }
 
-    /* This method turns a number of bytes into a human readable version
-     * with figs decimal places
-     */
-    public static String bytesToHumanReadable(long v, int figs) {
-        if (v < 1024) {
-            return v + " " + BundleUtil.getStringFromBundle("file.addreplace.error.byte_abrev");
-        }
-        // 63 - because long has 63 binary digits
-        int trailingBin0s = (63 - Long.numberOfLeadingZeros(v)) / 10;
-        //String base = "%."+figs+"f %s"+ BundleUtil.getStringFromBundle("file.addreplace.error.byte_abrev");
-        return String.format("%." + figs + "f %s" + BundleUtil.getStringFromBundle("file.addreplace.error.byte_abrev"), (double) v / (1L << (trailingBin0s * 10)),
-                             " KMGTPE".charAt(trailingBin0s));
-    }
+	/*
+	 * This method turns a number of bytes into a human readable version
+	 */
+	public static String bytesToHumanReadable(final long v) {
+
+		return bytesToHumanReadable(v, 1);
```

Comment:
I don't think there's a style config in the repo, but like default IDE style configuration (at least in intelliJ) we use 4 spaces for tabs throughout the codebase.